Userland/Libraries/LibCore/DateTime.cpp

```diff
@@ -446,25 +435,19 @@ Optional<DateTime> DateTime::parse(StringView format, DeprecatedString const& st
             break;
         }
         case 'n':
-        case 't':
-            while (is_ascii_blank(string[string_pos])) {
-                string_pos++;
-            }
-            break;
-        case 'p': {
-            auto ampm = string.substring_view(string_pos, 2);
-            if (ampm == "PM" && tm.tm_hour < 12) {
-                tm.tm_hour += 12;
-            }
-            string_pos += 2;
+        case 't': {
```

Comment:
Let's not add curly braces to case labels where they aren't needed, here and below.